git-clone/entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+#
+# The entrypoint removes the previous mount-path and does a fresh
+# checkout of the repository. It also initializes git lfs and sets
+# the proper file permissions.
+set -x
+
+if [ "$LFS_AUTO_FETCH" = 1 ]; then LFS_SKIP_SMUDGE="";
+else LFS_SKIP_SMUDGE="--skip-smudge";
+fi
+
+rm -rf ${MOUNT_PATH}/*
+(rm -rf ${MOUNT_PATH}/.* || true)
+git lfs install $LFS_SKIP_SMUDGE --system
+git clone $REPOSITORY ${MOUNT_PATH}
+git lfs install $LFS_SKIP_SMUDGE --local
```

Comment:
can we use `GIT_LFS_SKIP_SMUDGE` which should work without using option?

Reply:
true, that would be better

Reply:
Actually I changed my mind -- the reason for doing it this way is that it gets baked into the repository configuration. So I would leave it as-is.

Reply:
I don't get why having it baked into the repository is better.

Reply:
because this is a volume that is shared between containers. We then have to pass the environment variable to all the relevant containers, which leaves more room for error. Here we just set it once. I know that passing an environment variable is not the end of the world, but errors with LFS configuration lead to potentially catastrophic repository... "entanglements" :)